db/migrate/20220221181805_integer_fields.rb

```diff
@@ -0,0 +1,83 @@
+class IntegerFields < ActiveRecord::Migration[7.0]
+  def up
+    change_table :case_logs, bulk: true do |t|
+      t.remove :sex1
+      t.column :sex1, :integer
```

Comment:
Not an integer? Isn't it like "F, M, X ..."

Reply:
Done